src/test/java/com/exactpro/junit5/WithByteBufInvocationContext.java

```diff
@@ -0,0 +1,61 @@
+package com.exactpro.junit5;
+
+import io.netty.buffer.ByteBuf;
+import org.junit.jupiter.api.extension.Extension;
+import org.junit.jupiter.api.extension.TestTemplateInvocationContext;
+
+import java.util.Collection;
+import java.util.List;
+
+import static java.util.Collections.singletonList;
+
+public class WithByteBufInvocationContext implements TestTemplateInvocationContext {
+
+    private final List<ByteBuf> arguments;
+
+    private StringBuilder displayName;
```

Comment:
Don't store it in class. There are no guarantee that getDisplayName is called only once.